README.md

```diff
@@ -267,16 +267,18 @@ Activate your virtual environment. On Unix systems, this can be done with the fo
 source env/bin/activate
 ```
 
+<a name="env"></a>
```

Comment:
I like the idea of adding an anchor here. Shall we do this for other parts of the README in the future as well?

Reply:
I believe that the only information that was really removed, is the Windows-specific commands for setting environment variables. I don't think that the goal of our README is to teach people how to use different operating systems. At most, I would add a external link to some additional documentation for people who need it.

And yes, I think that there can be other good use cases for internal links in the README was well, feel free to add more.

Reply:
Okay understood!